nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -848,136 +768,4 @@ phNciNfc_ResetNtfCb(void*     pContext,
     }
     PH_LOG_NCI_FUNC_EXIT();
     return wStatus;
-}
-
-static NFCSTATUS phNciNfc_RegAllNtfs(void*     pContext)
```

Comment:
phNciNfc_RegAllNtfs is not called from phNciNfc_Init.c anymore after the change.
I through it was making more sense to move it where it is used.

Do you want that functions back in phNciNfc_Init.c despite that fact ?